### DISCUSSION TITLE
Online RAVE Workshop @BEK

### DISCUSSION THREAD

**Discussion by materialvision** [acids-ircam/RAVE#69] [General]
MACHINE LEARNING FOR SOUND MAKERS – RAVE
BEK invites participants to a new online workshop on the topic of machine learning for sound makers, as part of our collaboration series with Notam: “Workshops for Advanced Users”.

Workshop dates: 20–24 June

Registration deadline: 31 May 

This workshop will focus on the recently developed software / machine learning model RAVE (Realtime Audio Variational autoEncoder), and is led by the main developer Antoine Caillon, part of the ACIDS group (“Artificial Creative Intelligence and Data Science”) at Ircam in Paris.

more info:
https://bek.no/en/workshop-machine-learning-for-sound-makers-rave/

**Comment by JulienVincenot** [comment]
Hi @caillonantoine 
By any chance are there any traces of that workshop?
Otherwise any plans for another workshop this summer?
Merci !
Julien

**Reply by caillonantoine** [reply]
You can join the [discord server](https://discord.gg/m6R3WTsn) associated with this workshop, I think there are some material left there :)

**Reply by JulienVincenot** [reply]
great thanks Antoine !

**Reply by plagasul** [reply]
would it be possible to obtain a new invitation to the Discord, or any other access to the materials, please? The invitation from Jan 25 2023 is not valid anymore.

Thank you very much

**Comment by materialvision** [comment]
Hi try this: https://discord.gg/4EGMDtcFUn

Espen
